feature-libs/cart/import-export/core/config/import-export-config.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import { Config } from '@spartacus/core';
+import { FileValidity } from '../model/import-export.model';
+
+@Injectable({
+  providedIn: 'root',
+  useExisting: Config,
+})
+export abstract class ImportExportConfig {
+  importExport: {
```

Comment:
I'm sorry I was not enough precise before. The `importExport` key is a bit too general.  What about `cartImportExport`?